apps/prairielearn/src/pages/instructorQuestionSettings/instructorQuestionSettings.html.ts

```diff
@@ -77,323 +77,266 @@ export function InstructorQuestionSettings({
+                </small>
+              </div>
+              <div class="form-group">
+                <label for="qid">QID</label>
+                ${
+                  resLocals.authz_data.has_course_permission_edit &&
+                  !resLocals.course.example_course
+                    ? html`
+                        <button
+                          type="button"
+                          class="btn btn-xs btn-secondary mr-2"
+                          id="changeQidButton"
+                          data-toggle="popover"
+                          data-container="body"
+                          data-html="true"
+                          data-placement="auto"
+                          title="Change QID"
+                          data-content="${renderEjs(
+                            __filename,
+                            "<%= include('../partials/changeIdForm') %>",
+                            {
+                              id_label: 'QID',
+                              buttonID: 'changeQidButton',
+                              id_old: resLocals.question.qid,
+                              ids: qids,
+                              ...resLocals,
+                            },
+                          )}"
+                          data-trigger="click"
+                        >
+                          <i class="fa fa-i-cursor"></i>
+                          <span>Change QID</span>
+                        </button>
+                      `
+                    : ''
+                }
+                ${
+                  questionGHLink
+                    ? html`<a target="_blank" href="${questionGHLink}"> view on GitHub </a>`
+                    : ''
+                }
+                <input
+                  type="text"
+                  class="form-control"
+                  id="qid"
+                  name="qid"
+                  value="${resLocals.question.qid}"
+                  disabled
+                />
+                <small class="form-text text-muted">
+                  This is a unique identifier for the question. (e.g., "addNumbers")
+                </small>
+              </div>
+              <div class="form-group">
+                <label for="type">Type</label>
+                <input
+                  type="text"
+                  class="form-control"
+                  id="type"
+                  name="type"
+                  value="${resLocals.question.type}"
+                  disabled
+                />
+                <small class="form-text text-muted"> This is the type of question. </small>
+              </div>
+              <div class="form-group">
+                <div>Topic</div>
+                <div name="topic">
+                  ${renderEjs(__filename, "<%- include('../partials/topic') %>", {
+                    topic: resLocals.topic,
+                  })}
+                </div>
+              </div>
```

Comment:
I don't think the `form-group` class will do much to help us here, and setting `name="..."` on a `div` won't do anything at all.

For better accessibility, I'd recommend using heading tags (`<h1>`, `<h2>`, etc.) as appropriate for these sections. Here's a primer if you're new to those: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements

Reply:
Updated these to use `<h2>` for these sections in [f9945bf](https://github.com/PrairieLearn/PrairieLearn/pull/9687/commits/f9945bf0e4c4e0d20d0a34f07c433d9e135dc0c7). The display of `<h2>` was quite large for these subheaders so added `class="h5"` to reduce the text to a more appropriate size.